.github/renovate.json5

```
@@ -66,6 +67,18 @@
   ],
   separateMinorPatch: true,
   packageRules: [
+    {
+      // automerge patch updates
+      matchUpdateTypes: ['patch'],
+      automerge: true,
+    },
```

We might consider adding the following line:
```suggestion
    {
      // automerge patch updates
      matchUpdateTypes: ['patch'],
      matchCurrentVersion: '!/^0/',
      automerge: true,
    },
```
As suggested by the docs: https://docs.renovatebot.com/key-concepts/automerge/#automerge-non-major-updates

This avoids automerging patch updates of `0.x.x` dependencies.

---

I think automerging patch updates of v0 dependencies like `k8s.io/*` and `sigs.k8s.io/controller-runtime` is valid and what we want. Your suggestion would exclude automerging of these cases.

Later on, we could consider adding automerge of minor updates for non-v0 dependencies, similar to this: https://github.com/timebertt/kubernetes-controller-sharding/blob/b010049123bf440e1e262ac2dc82edaf0485101c/.github/renovate.json5#L47-L58
I had good experiences with this, but milage may vary in this repository. So I'd suggest looking into this in a later step.

---

That's fine with me :) Thanks for the explanation